src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -2056,6 +2056,7 @@ MulticopterPositionControl::task_main()
 
 		} else {
 			reset_yaw_sp = true;
+			_att_sp.yaw_sp_move_rate = 0.0f;
```

Comment:
Or even better, move the logic into a reset_yaw_sp() function and call it [there](https://github.com/PX4/Firmware/pull/4921/files#diff-fc77c3ef569029d45764664c75d4b0c1R1961) like reset_pos/alt_sp